pkgs/by-name/do/door-knocker/package.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchFromGitea
+, blueprint-compiler
+, desktop-file-utils
+, glib
+, gtk4
+, libadwaita
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook4
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "door-knocker";
+  version = "0.4.1";
```

Comment:
New version is available: https://codeberg.org/tytan652/door-knocker/compare/0.4.1...0.4.2